webpack.config.js

```diff
@@ -25,13 +26,9 @@ module.exports = function(env = {}) {
       chunkFilename: "[name].css",
     }),
     // new HardSourceWebpackPlugin(),
-    // GoldenLayout requires these libraries to be available in
-    // the global scope
     new webpack.ProvidePlugin({
       React: "react",
       ReactDOM: "react-dom",
-      $: "jquery",
-      jQuery: "jquery",
```

Comment:
Sorry, I'm only realizing it now: I think the entire `ProvidePlugin` stuff can be removed. From the comment above it seems like even react and react-dom only had to be provided because of GoldenLayout.

Reply:
Sure, no problemo